kani-driver/src/args_toml.rs

```diff
@@ -106,7 +106,8 @@ fn toml_to_args(tomldata: &str) -> Result<(Vec<OsString>, Vec<OsString>)> {
     for (flag, value) in map {
         if flag == "cbmc-args" {
             // --cbmc-args has to come last because it eats all remaining arguments
-            insert_arg_from_toml(&flag, &value, &mut cbmc_args)?;
+            cbmc_args.push("--cbmc-args".into());
+            cbmc_args.append(&mut cbmc_arg_from_toml(&value)?);
         } else {
             insert_arg_from_toml(&flag, &value, &mut args)?;
```

Comment:
Why is the new function needed? Why aren't you using it in the `else` branch?

Reply:
That's a great question. The problem here is that `--cbmc-args` will be the only case where we won't replicate the flag name when flattening an array. Here is an example:

```toml
[package.metadata.kani.flags]
package = ["p1", "p2"]
harness = ["foo", "bar"]
cbmc_args = ["--outfile", "filename"]
```

We will convert this into these args:
```rust
["--package", "p1", "--package", "p2", "--harness", "foo", "--harness", "bar", "--cbmc-args", "--outfile", "filename"]
```

Does that make sense?

Reply:
Yes, thank you!